File: ws-vs/Primeiro/retangulo.cs
using System;
using System.Globalization;

namespace Programa {
    class Program {
        static void Main(string[] args) {

            int base, altura, diagonal, area, perimetro;

            Console.Write("Digite o valor da base do retangulo : ");
            base = int.parse(console.readline());

            Console.Write("Digite o valor da altura do retangulo : ");
            altura = int.parse(console.readline());

            area = base * altura;
            perimetro = (2 * base) + (2 * altura);
            diagonal = base ^1/2;

            Console.Write("Valor da aree é  : " + area);
            Console.Write("Valor do perimetro é  : " + perimetro);
            Console.Write("Valor da diagonal é  : " + diagonal);

        }
    }
}

File: ws-eclipse/projetos_udemy/curso_udemy/src/retangulo.java

import java.util.Scanner;

public class Main {
public static void matriz (String[] args) {
	
 Scanner sc = new Scanner(System.in);
 
 int base, altura, diagonal, area, perimetro;
 
 System.out.println("Digite a base do retangulo : ");
 base = sc.nextInt();
 
 System.out.println("Digite a altura do retangulo : ");
 altura = sc.nextInt();
 
 area = base * altura;
 perimetro = (2 * base) + (2 * altura);
 diagonal = (2 * base^1/2);
 
 System.out.println("A area é : " + area);
 System.out.println("O perimetro é : " + perimetro);
 System.out.println("A diagonal é : " + diagonal);
 
}
}
 